src/webapi/element.rs

```diff
@@ -163,8 +164,47 @@ pub trait IElement: INode + IParentNode + IChildNode {
     fn has_pointer_capture( &self, pointer_id: i32 ) -> bool {
         js!( return @{self.as_ref()}.hasPointerCapture( @{pointer_id} ); ).try_into().unwrap()
     }
+
+    /// Insert nodes from HTML fragment into specified position.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML)
+    // https://dom.spec.whatwg.org/#ref-for-dom-element-insertadjacentelement
+    fn insert_adjacent_html( &self, position: &InsertPosition, html: &str ) -> Result<(), InsertAdjacentError> {
```

Comment:
Since the enum is `Copy` this should take `InsertPosition`. (In general the `Copy` types should be passed by value while `Clone` types should be passed by reference.)

Reply:
Good point, fixed.